src/doc/gui_manual/Operators/OperatorTypes/PICS/index.rst

```diff
@@ -0,0 +1,268 @@
+J. of Computational Science, 7 (2015) 26-36
+<http://georgehaller.com/reprints/lcstool.pdf>`_
+
+.. toctree::
+    :maxdepth: 1
+
+    IntegralCurve_operator
+    LCS_operator
+    LimitCycle_operator
+    Poincare_operator
+
+Parameters
+----------
+
+Common to all PICS operators is a four tab GUI: Source, Integration,
+Appearance, and Advanced (the Poincare operator also has an Analysis tab).
+These tabs contain many functions that are common across all four operators.
+The following is a description of those common features. Please note: on a mac
+system, the contents of the Source and Integration tabs are combined into a
+single Integration tab.
+
+
+Source
+^^^^^^
+
+The set of points to seed the integral curves in the operator. See specific
+operator for details.
+
+Field
+"""""
+
+Sets the field type so that the native elements are used when interpolating the
+vector fields. Each operator provides the following options:
+
+Default
+    Use VisIt's native VTK mesh structure performing linear interpolation on
+    the vector field.
+    
+Flash
+    Evaluates the velocity field via the Lorentz force. Parameters are:
+
+    * Constant - A constant multiple applied to the velocity.
+    * Velocity - When combined with Leap-Frog integration, this sets the
+    initial velocity used in the integration.
+   
+M3D-C1 2D
+    Evaluates the 3D magnetic field via a 2D poloidal 6th order polynomial.
+    Parameters are:
+
+    * Constant - A constant multiple applied to the perturbed part of the
+      field.
+   
+M3D-C1 3D
+    Evaluates the 3D magnetic field via a 2D poloidal 6th order polynomial and
+    1D toroidal 4th order Bezier spline.
+    
+Nek5000
+    Evaluates the 3D vector field using `Nek5000 <https://nek5000.mcs.anl.gov/>`_
+    spectral elements.
+    
+Nektar++
+    Evaluates the 3D vector field using `Nektar++ <https://www.nektar.info/>`_
+    spectral elements.
+        
+NIMROD
+    Evaluates the 3D magnetic field via a 2D poloidal 3rd order Lagrange-type
+    element and 1D toroidal finite Fourier series element (future
+    implementation).
+
+
+Integration
+^^^^^^^^^^^
+
+Specify settings for the numerical integrator. See each operator for varied
+settings.
+
+
+Integrator
+""""""""""
+
+Sets the integration scheme. Options are:
+
+Forward Euler
+    The basic Euler integration with a fixed step length. This integrator is
+    not recommended.
+
+Leap Frog
+    Simple method for numerical integration where both the position and
+    velocity are updated using a fixed step length.
+
+Runge-Kutta-Dormand-Prince (RKDP)
+    A Runge-Kutta based scheme with adaptive step size control.
+    
+Adams-Bashforth (Multi-step)
+    A linear multistep integration using a fixed step length.
+    
+Runge-Kutta
+    Traditional 4th order Runge-Kutta with a fixed step length.
+    
+M3D-C1 2D Integrator (M3D-C1 2D fields only)
+    Uses Newton's method under the hood.
+    
```

Comment:
Side note but reading this makes me wonder if whether [SUNDIALS](https://computing.llnl.gov/projects/sundials) make make for a useful option here? Again, maybe a question for @ARSanderson

Reply:
The fusion folks ofter do funny integration because of their elements - this is a good case in point.

Using an external pkg for the integration is typically not a payoff because of the format conversion, thus we roll our own. This becomes a bigger issue trying to handle higher order elements.